Explorer/Assets/DCL/CharacterPreview/CharacterPreviewContainer.cs

```diff
@@ -0,0 +1,24 @@
+using UnityEngine;
+
+namespace DCL.CharacterPreview
+{
+    /// <summary>
+    ///     Contains serialized data only needed for the character preview
+    ///     See CharacterPreviewController in the old renderer
+    /// </summary>
+    public class CharacterPreviewContainer : MonoBehaviour
```

Comment:
Considering that `CharacterPreviewContainer` should be pluggable freely to other parts this object can't belong to the View domain.
- It should be represented by a separate prefab that encapsulates the scene with camera, parent, and whatever is needed
- Preferably these scenes should be poolable if we expect any number can be instantiated
- Factory should use the pool to provide an instance